src/Simulation/Native/CMakeLists.txt

```diff
@@ -44,6 +44,9 @@ else (MSVC)
   set(CMAKE_CXX_FLAGS "-static-libgcc")
 endif (MSVC)
 
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /Qspectre")
```

Comment:
This seems unrelated to the links modification? If it's a necessary change, I'd recommend doing it separately, figuring out why the build broke after only fixing some links would be a pain :-)

Reply:
Reverted, that was my mistake. Picked up some changes from other work that shouldn't have been included here.